en_US/data-integration/data-bridge-mqtt.md

```diff
@@ -1,56 +1,71 @@
 
-- **Egress** mode
+### Ingress Mode
 
-    Forwarding messages from the current cluster to the bridged brokers following the rule settings.
-
-Below is the message flow in **ingress** direction:
+In ingress mode, the local EMQX subscribes to the topics from the bridged remote MQTT brokers and distributes the received messages within the current cluster. Below is the message flow in **ingress** direction:
 
 <img src="./assets/bridge_igress.png" alt="bridge_igress" style="zoom:50%;" />
 
-And this is the message flow in **egress** direction:
+The MQTT data bridge can be used either alone or in conjunction with rules for more powerful and flexible data processing capabilities. In **ingress** direction, the data bridge can be used as the data source of the rule. The message flow for MQTT data bridge working with rules is as follows:
+
+<img src="./assets/bridge_igress_rule_link.png" alt="bridge_igress_rule_link" style="zoom:50%;" />
+
+### Egress Mode
+
+In egress mode, the local EMQX forwards messages from the current cluster to the bridged remote MQTT brokers following the rule settings. And this is the message flow in **egress** direction:
 
 <img src="./assets/bridge_egerss.png" alt="bridge_egerss" style="zoom:50%;" />
 
-MQTT data bridge can be used either alone or in conjunction with rules for more powerful and flexible data processing capabilities.
+Similar to the ingress mode, the MQTT data bridge can also be used in conjunction with rules. In **egress** direction, the data bridge can be used as the action of the rule:
 
-The message flow for MQTT data bridge working with rules are as follows:
+<img src="./assets/bridge_egress_rule.png" alt="bridge_egress_rule" style="zoom: 50%;" />
 
-In **ingress** direction, it can be used as the data source of the rule:
+### Connection Pool
 
-<img src="./assets/bridge_igress_rule_link.png" alt="bridge_igress_rule_link" style="zoom:50%;" />
+EMQX allows more than one client to connect to the bridged MQTT broker at the same time. When creating the data bridge, you can set a pool of MQTT client connetions and configure the pool size indicating the number of the client connections in the pool. Enabling connection pool in MQTT data bridge allows for efficient utilization of server resources, leading to greater message throughput and improved concurrency performance. This brings benefits in scenarios with high loads and concurrent connections.
 
-In **egress** direction, it can be used as the action of the rule:
+Because the MQTT protocol requires that a client connects to one broker must has it's own unique client ID, each client in the connection pool is assigned to a unique client ID. To make the client ID predictable, EMQX automatically generates the Client ID following the pattern below:
```

Comment:
```suggestion
Because the MQTT protocol requires that a client connecting to the broker must have its own unique Client ID, EMQX automatically generates the Client ID for each client in the connection pool following the pattern below:
```